### PR TITLE
feat: add authentication

### PR DESCRIPTION
**Pull request by nerdyspook**
**Added authentication functionalities**
> Login functionality
> Signup functionality

### CODE REVIEW

**Review by devananth — APPROVED** [review]
- liked the way you **structured the files**
- Good work !

File: src/reset.scss [hunk 1]
```diff
@@ -103,7 +104,8 @@ input {
 }
 
 body {
-    margin-top: var(--header-height);
+    // margin-top: var(--header-height);
```
Comment (deepak29-git) [inline src/reset.scss:107]: remove commented code

**Review by deepak29-git — APPROVED** [review]
your folder structure is good and code is very clean

File: src/utilities/login-user.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
+
+export const loginUser = async (email, password, dispatchAuth, navigate) => {
+    try {
+        const response = await axios({
```
Comment (parthgabani777) [inline src/utilities/login-user.js:5]: you can use axios.post() directly.

**Review by parthgabani777 — APPROVED** [review]
Code looks well structured.

**Review by SanjayJatti — APPROVED** [review]
- Liked the way created separate utilities for log in user, add new user
- Code is clean

**Pull request merged**